README.md

```diff
@@ -1 +1,223 @@
-# notify-client-go
+# Notify Go client
+
+This documentation is for developers interested in using a Go client to integrate with Notify.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Getting started](#getting-started)
+- [Send messages](#send-messages)
+- [Tests](#tests)
+
+## Installation
+
+```shell
+go get -u github.com/govau/notify-client-go
+```
+
+## Getting started
+
+```Go
+client, err := notify.NewClient(api_key)
+```
+
+Generate an API key by logging in to [Notify.gov.au](https://notify.gov.au) and going to the _API integration_ page.
+
+## Send messages
+
+### Text message
+
+#### Method
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```go
+resp, err := client.SendSMS(
+  templateId,
+  phoneNumber,
+  notify.Reference("Sam's reminders"),
```

Comment:
Reference would usually be an internal reference number, so maybe better as more of an ID e.g. `1234`.

Reply:
I see that it can also be used for batches. Probably OK to leave it as you have.